### ISSUE TITLE
Restructure Chapter 3

### ISSUE THREAD

**Issue by HeinrichPet** [International-Data-Spaces-Association/IDS-RAM_4_0#112]
We want to write about Policy Negotiation and Policy Enforcement in chapter [3.3 Process Layer](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer). The goal is to write how Policy Negotiation between Connectors takes place, and how some policies start processes to get data from Policy Information Points, etc..

However, we see the problem that contracts are only described in the following chapter [3.4 Information Layer](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer). In the Information Layer also the Self-Description is defined, which is already used in the [3.3.2 Data Offering](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_2_Data_Offering.md) chapter.

Basically, we think that the Informaiton layer should be before the communication (in our case process) layer. Logically, we should always first define what we are communicating and then how you are doing it: https://de.wikipedia.org/wiki/Smart_Grid_Architecture_Model.

Any opinions to that?

@sebbader @ssteinbuss @juliapampus @Brandstaedter @clange

**Comment by ssteinbuss** [comment]
Closed by #193
